js/script.js

```diff
@@ -0,0 +1,67 @@
+const myBtn = document.querySelector('.my-btn');
+
+const eachSpeakers = [
+  {
+    name: 'Ezra Ermy',
+    picture: 'images/ez.JPG',
+    profession: 'Ezra is a full-stack web developer and researcher',
+    backgound: 'Excellent knowledge of JavaScript/TypeScript, HTML and CSS. Provides mentorship and supervise developers',
+  },
+  {
+    name: 'Abenu Ermy',
+    picture: 'images/ez.JPG',
+    profession: 'Ezra is a full-stack web developer and researcher',
+    backgound: 'Excellent knowledge of JavaScript/TypeScript, HTML and CSS. Provides mentorship and supervise developers',
+  },
+  {
+    name: 'Ekut Ermy',
+    picture: 'images/ez.JPG',
+    profession: 'Ezra is a full-stack web developer and researcher',
+    backgound: 'Excellent knowledge of JavaScript/TypeScript, HTML and CSS. Provides mentorship and supervise developers',
+  },
+  {
+    name: 'Tutu Ermy',
+    picture: 'images/ez.JPG',
+    profession: 'Ezra is a full-stack web developer and researcher',
+    backgound: 'Excellent knowledge of JavaScript/TypeScript, HTML and CSS. Provides mentorship and supervise developers',
+  },
```

Comment:
- [ ] I think it would be better if you add 2 more speakers to match the Behance template design